<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolboxBase</class>
 <widget class="QWidget" name="ToolboxBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>563</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>1</number>
   </property>
   <property name="spacing">
    <number>1</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QToolBox" name="box">
       <widget class="QWidget" name="pgEntries">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>343</width>
          <height>437</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Blog Posts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="KListWidget" name="lstEntriesList">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="isWrapping" stdset="0">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="btnEntriesUpdate">
             <property name="toolTip">
              <string>Update entries list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KPushButton" name="btnEntriesClear">
             <property name="toolTip">
              <string>Clear entries list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KSeparator" name="kseparator">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KPushButton" name="btnEntriesRemove">
             <property name="toolTip">
              <string>Remove selected entry from server</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
        <zorder>lstEntriesList</zorder>
        <zorder></zorder>
       </widget>
       <widget class="QWidget" name="pgCat">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>343</width>
          <height>437</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Post &amp;Categories</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="frameCat">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>333</width>
              <height>362</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="margin">
              <number>2</number>
             </property>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="KPushButton" name="btnCatReload">
               <property name="toolTip">
                <string>Reload categories list</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KPushButton" name="btnCatAdd">
               <property name="toolTip">
                <string>Add a new category to the selected blog</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lblCatTags">
               <property name="text">
                <string>&amp;Tags:</string>
               </property>
               <property name="buddy">
                <cstring>txtCatTags</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="KLineEdit" name="txtCatTags"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgOptions">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>343</width>
          <height>437</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Post &amp;Options</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string comment="Status of the current post">Status:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="KComboBox" name="comboOptionsStatus">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Published</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Draft</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Local Draft</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Feedback:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QCheckBox" name="chkOptionsComments">
               <property name="text">
                <string>Comments</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="chkOptionsTrackback">
               <property name="text">
                <string>Trackback</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="chkOptionsTime">
           <property name="text">
            <string>Modify time&amp;stamp</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>15</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QFrame" name="frame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0" colspan="3">
               <widget class="KDateWidget" name="optionsDate">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QTimeEdit" name="optionsTime">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="buttonSymbols">
                 <enum>QAbstractSpinBox::UpDownArrows</enum>
                </property>
                <property name="correctionMode">
                 <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="2">
               <widget class="KPushButton" name="btnOptionsNow">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Now</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Permalink (Wordpress):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="txtSlug"/>
         </item>
         <item>
          <widget class="QLabel" name="lblOptionsTrackBack">
           <property name="text">
            <string>&amp;Trackback URLs:</string>
           </property>
           <property name="buddy">
            <cstring>txtOptionsTrackback</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="KLineEdit" name="txtOptionsTrackback"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Summary:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="txtSummary">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pgNotes">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>343</width>
          <height>437</height>
         </rect>
        </property>
        <attribute name="label">
         <string>&amp;Local Entries</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="margin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTableWidget" name="localEntriesTable">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="showDropIndicator" stdset="0">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <column>
            <property name="text">
             <string comment="Post title">Title</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Blog</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="KPushButton" name="btnLocalRemove">
             <property name="toolTip">
              <string>Remove selected entry</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_10">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KDateWidget</class>
   <extends>QWidget</extends>
   <header>kdatewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnCatAdd</tabstop>
  <tabstop>txtCatTags</tabstop>
  <tabstop>chkOptionsTime</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>chkOptionsTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>optionsTime</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOptionsTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>btnOptionsNow</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkOptionsTime</sender>
   <signal>toggled(bool)</signal>
   <receiver>optionsDate</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
